src/famfs_lib.c

```diff
@@ -3715,8 +3772,16 @@ famfs_mkfs(const char *daxdev,
 	 * If the role is FAMFS_CLIENT, they'll have to manually blow away the superblock
 	 * if they want to do a new mkfs.
 	 */
-	if (rc == FAMFS_CLIENT)
+	if (rc == FAMFS_CLIENT) {
+		fprintf(stderr, "Mkfs failed, not allowed on client nodes.\n");
```

Comment:
A better error message would be "Error: Device %s has a superblock that is owned by another host\n" (or something along those lines)

Reply:
Changed.